src/styles/stars.css

```
@@ -0,0 +1,22 @@
+.stars--4 {
```

```suggestion
.stars {
```

it should be for stars block in general, not only for block with 4 stars